OasisDEX/oasis-borrow
# Swap widget predefining variant

# Swap widget predefining variant

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added handling for initializing swap widget with predefined variant action (swap / bridge)
  
## How to test 🧪
  <Please explain how to test your changes>

- using new navigation, try to open widget by click on swap or bridge, in both cases widget should be opened with proper action
